frontend/app/src/main/java/com/example/speechbuddy/compose/home/HomeScreen.kt

```diff
@@ -0,0 +1,143 @@
+@Composable
+fun HomeScreen(
+
+) {
+    val navController = rememberNavController()
+    Scaffold(
+        bottomBar = {
+            BottomNavigationBar(
+                items = listOf(
+                    BottomNavItem(
+                        "SymbolSelection",
+                        "symbolselection",
+                        R.drawable.outline_touch_app_24
+                    ),
+                    BottomNavItem(
+                        "TTS",
+                        "tts",
+                        R.drawable.outline_volume_up_24
+                    ),
+                    BottomNavItem(
+                        "CreateSymbol",
+                        "createsymbol",
+                        R.drawable.outline_add_a_photo_24
+                    ),
+                    BottomNavItem(
+                        "Settings",
+                        "settings",
+                        R.drawable.outline_settings_24
+                    )
+                ),
+                navController = navController,
+                modifier = Modifier
+                    .padding(horizontal = 0.dp, vertical = 10.dp)
+                    .height(100.dp),
+                onItemClick = {
+                    navController.navigate(it.route)
+                }
+            )
+        }
+    ) {
+        HomeScreenNavHost(
+            navController = navController
+        )
+    }
+}
+
+@Composable
+fun HomeScreenNavHost(
+    navController: NavHostController
+) {
+    NavHost(navController = navController, startDestination = "symbolselection") {
+        composable("symbolselection") {
+            SymbolSelectionScreen()
+        }
+        composable("tts") {
+            TextToSpeechScreen()
+        }
+        composable("createsymbol") {
+            SymbolCreationScreen()
+        }
+        composable("settings") {
+            SettingsScreen()
+        }
+    }
+}
+
+@Composable
+fun BottomNavigationBar(
+    items: List<BottomNavItem>,
+    navController: NavController,
+    modifier: Modifier = Modifier,
+    onItemClick: (BottomNavItem) -> Unit
+) {
+    val backStackEntry = navController.currentBackStackEntryAsState()
+    NavigationBar(
+        modifier = modifier,
+        containerColor = MaterialTheme.colorScheme.secondaryContainer
+    ) {
+        items.forEach { item ->
+            val selected = item.route == backStackEntry.value?.destination?.route
+            NavigationBarItem(
+                selected = selected,
+                onClick = { onItemClick(item) },
+                icon = {
+                    Icon(
+                        painter = painterResource(id = item.icon),
+                        contentDescription = null
+                    )
+                },
+                colors = NavigationBarItemDefaults.colors(
+                    selectedIconColor = MaterialTheme.colorScheme.onPrimaryContainer,
+                    unselectedIconColor = MaterialTheme.colorScheme.secondary
```

Comment:
```suggestion
                    unselectedIconColor = MaterialTheme.colorScheme.outline
```

이거 outline으로 하면 색 대비가 더 잘 보일 것 같아요!